### DISCUSSION TITLE
Tree style tabs

### DISCUSSION THREAD

**Discussion by SomeGuyNamedMay** [raphamorim/rio#283] [Ideas]
what are your thoughts about implementing something like tree style tabs?

**Comment by raphamorim** [comment]
Hey @SomeGuyNamedMy thanks for the issue,

Regarding tree style tabs what you think about the first level would be the window, second level tabs and third level panels?

**Comment by SomeGuyNamedMay** [comment]
by tree style tabs i am refering to something like this
![image](https://github.com/raphamorim/rio/assets/83663766/b5f00ba2-7e72-4f1c-bb2b-d839b4ecd461)
were each tab can have a number of tabs under it, but the first "layer" behaves like normal tabs

**Comment by raphamorim** [comment]
I think that would be great, isn't part of my roadmap now but if anyone would want to add support to it on Rio I gladly can help with it.
